docs/Tutorial_MPDist.ipynb

```diff
@@ -0,0 +1,380 @@
+{
```

Comment:
<p>*of their subsequences</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/433/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Instead of "determining whether most" maybe it should be "determining whether a limited subset of subsequences - as parameterized by a threshold - are similar"</p>

Reply:
<p> </p><blockquote>For instance, if two time series is made up of the same repeating subsequences of window length <code>m</code>, then they would MPdist of 0 (if window size for MPdist is set to <code>m</code>), even if they are phase shifted. On the other hand, the Euclidean distance would be non-zero as long as they are phase shifted. </blockquote><p>I find this sentence to be very confusing/abstract as it is not anchored to anything and expects the user to already have some prior knowledge</p>

Reply:
<p><em>For instance, if two time series <s>is</s> </em><strong><em>are</em></strong><em> made up of the same repeating subsequences of window length </em><code><em>m</em></code><em>, then they would </em><strong><em>have an</em></strong><em> MPdist of 0...</em></p><p> </p><p>I agree it's a bit difficult to follow this definition, although it's tough to come up with an alternative phrasing. Maybe something like: <em>For instance, two time series that are made up of the same periodic subsequences, but are phase shifted, their MPdist would be zero, while their Euclidean distance would be non-zero.</em></p><p> </p><p>I understand the significance of <code>m</code> , but maybe in this exact definition you can remove the mention of it? Idk it seems cleaner for the sake of explaining.</p>